### ISSUE TITLE
Assignment 2 of Udacity course

### ISSUE THREAD

**Issue by ming030890** [tensorflow/tensorflow#857]
Lecture one covers logistic regression and stochastic gradient descent. Assignment 2, assignment of this lecture, requests students to implement a neural network which is covered in later chapter.
I suggest the question should be related to the topic of lecture.

**Comment by napratin** [comment]
Thanks for pointing it out!

The first part of Assignment 2 (code provided) is relevant immediately after SGD in L1, but the stated problem requires knowledge of ReLUs and the ability to create a multi-layer network. So Assignment 2 has been moved to right after Backprop in L2, and just before discussing deeper networks.
